oximeter/oximeter/schema/dendrite.toml

```diff
@@ -53,6 +53,30 @@ description = "Revision number of the switch being managed"
 type = "uuid"
 description = "ID of the switch being managed"
 
+[fields.switch_fab]
+type = "string"
+description = "Fabrication plant identifier of the switch the link is on"
+
+[fields.switch_lot]
+type = "string"
+description = "Lot number of the switch the link is on"
+
+[fields.switch_wafer]
+type = "u8"
+description = "Wafer number of the switch the link is on"
+
+[fields.switch_wafer_loc_x]
+type = "i8"
+description = "X-coordinate wafer location of the switch the link is on"
```

Comment:
Do these have units? I doubt the Tofino software is documented enough, but if we do have them, could we put that in the description?

Reply:
I couldn't find this in bf_sde, but maybe @rcgoodfellow knows?

Reply:
Nothing I could find btw.

Reply:
I'm not aware of a unit for this quantity.